src/cache.cc

```diff
@@ -71,6 +72,10 @@ CACHE::mshr_type CACHE::mshr_type::merge(mshr_type predecessor, mshr_type succes
     }
   }
```

Comment:
Should these lines be moved into the debug listener?

Reply:
Yes, my intention was for the debug listener to replace the existing debug print statements. Once I clean those out and update the pull request, I'll resolve this conversation